concordium-sdk/src/main/java/com/concordium/sdk/transactions/GTUAmount.java

```diff
@@ -23,6 +25,10 @@ byte[] getBytes() {
         return value.getBytes();
     }
 
+    public static GTUAmount fromBytes(ByteBuffer source) {
```

Comment:
Does it have to be public? Otherwise I think just having `static GTUAmount fromBytes(ByteBuffer source)` is to prefer.

Reply:
But how is somebody to parse amounts otherwise?